interpreter/evalstate.go

```diff
@@ -36,35 +36,35 @@ type EvalState interface {
 type MutableEvalState interface {
 	EvalState
 
-	// SetRuntimeExpressionId sets the runtimeId for the given exprId.
-	SetRuntimeExpressionId(exprId int64, runtimeId int64)
+	// SetRuntimeExpressionID sets the runtime id for the given expr id.
```

Comment:
Do we want to capitalize 'id' here as well?

Reply:
Yes, good catch.